packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/PaymentMethods/CheckmoMethod/checkmoMethod.gql.js

```diff
@@ -0,0 +1,26 @@
+import gql from 'graphql-tag';
+// Sets the provided payment method object on the cart.
+export const SET_PAYMENT_METHOD = gql`
+    mutation setPaymentMethodOnCart(
+        $cartId: String!
+        $method: PaymentMethodInput!
```

Comment:
This is duplicated gql from the other components that set payment method. We could define a callback in the parent `paymentMethods` for setting this value and then thread it to children. 

I'm leaning towards just doing that.